201-api-management-logs-to-moesif-using-eventhub-webapp/azuredeploy.json

```diff
@@ -0,0 +1,324 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "moesifApplicationId": {
+            "type": "string",
+            "minLength": 50,
+            "metadata": {
+                "description": "Your Moesif Application Id (aka Collector Application Id) can be found in the Moesif Portal. After signing up for a Moesif account, your Moesif Application Id will be displayed during the onboarding steps. Sets environment variable APIMEVENTS-MOESIF-APPLICATION-ID in App Service"
+            }
+        },
+        "existingApiMgmtResourceGroup": {
+            "type": "string",
+            "defaultValue": "",
```

Comment:
use resourceGroup().location as the defaultValue and remove any conditional

Reply:
Thx for catching this. Updating this shortly. Actually it should be resourceGroup().name as template expects that.